libdevcore/JsonUtils.h

```diff
@@ -24,4 +24,25 @@ namespace dev
 {
 // Throws UnknownField() if _obj contains field names not listed in _allowedFields.
 void validateFieldNames(json_spirit::mObject const& _obj, std::set<std::string> const& _allowedFields);
+
+// Converts json value type to string
+std::string jsonTypeAsString(json_spirit::Value_type _type);
+
+enum jsonField
```

Comment:
`enum class` is preferable to `enum`. Also start type names with capital letter. Also this should be better named something like `JsonFieldPresence` maybe